components/page-header.js

```diff
@@ -0,0 +1,8 @@
+class PageHeader {
+  constructor(headerElement) {
+    this.headerElement = headerElement;
+  }
+  updateAverage(newAverage) {
+    gradeAverage.textContent = newAverage;
```

Comment:
You should be using the `this.headerElement` property here, not the variable you made to pass in the element to the class 👍